packages/bff/src/services/catalogService.ts

```diff
@@ -119,8 +205,89 @@ export function catalogServiceBuilder(
           totalCount: eservicesResponse.totalCount,
         },
       };
-
       return response;
     },
+    getProducerEServiceDescriptor: async (
+      eServiceId: string,
+      descriptorId: string,
```

Comment:
I see that later you are unsafeBranding these.
If we want to brand these params, maybe I would do it in the router and pass to this method already branded params. It's the same pattern we use in the `*- process` services so that we keep the unsafe branding only near the frontier, just after parsing/validating request body/params, and not inside the business logic. I would also do the same with the requesterId and add it as param.

```suggestion
      eServiceId: EServiceId,
      descriptorId: DescriptorId,
      requesterId: TenantId,
```

Reply:
This [commit](https://github.com/pagopa/interop-be-monorepo/pull/712/commits/480d43323060adfdab81dc03680c89c57c4e983f) limits `unsafeBrandId` usages to the router. However, it still remains where we have a type coming from the catalog API (I prefer to avoid adding an extra parameter solely for error message formatting).

Reply:
This [commit](https://github.com/pagopa/interop-be-monorepo/pull/712/commits/7dc73a20f75f65a16e938f00100ea8da9b22919f#diff-8fbd86b24fcfe096e08e400eed3d229b8a8cfa1848e0837ce7d3a940ba687149) remove branded type for error message.